src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,74 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+
+  &.open {
+    width: 360px;
+  }
+
+  .rs-base-layer-switch-layers {
+    display: flex;
+    overflow: auto;
+    height: 100%;
+    width: 360px;
+
+    .rs-base-layer-switch-button {
+      position: relative;
+      height: 85%;
+      min-width: 100px;
```

Comment:
i think putting a fixed with make more sense

Reply:
I only left the min-width